assets/styles.css

```diff
@@ -0,0 +1,189 @@
+/* styles resets */
+* {
+  text-decoration: none;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: 'Gothic A1', sans-serif;
+}
+
+/* responsiveness */
+
+.resize {
+  min-width: 1440px;
+  padding: 0 1px 0 1px;
+}
+
+/* Grid background and images */
+
+.grid_area {
+  background: url("../assets/images/background.jpg") 50% 100% no-repeat;
+  height: 538px;
+  max-width: 1440px;
+  display: flex;
+  flex-direction: column;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/* navbar */
+
+nav {
+  display: flex;
+  align-self: center;
+  margin-top: 83px;
+  width: 980px;
+}
+
+nav a {
+  background: linear-gradient(0deg, rgba(120, 120, 120, 1) 0%, rgba(95, 95, 95, 1) 50%, rgba(110, 110, 110, 1) 51%, rgba(120, 120, 120, 1) 100%);
+  border: 1px solid #757575;
+  color: white;
+  text-align: center;
+  line-height: 30px;
+  text-shadow: 0 0.6px rgba(0, 0, 0, 0.6), 0 -0.6px rgba(0, 0, 0, 0.9);
+  width: 106px;
+  height: 36px;
+  font-weight: bold;
+  font-size: 14px;
+  padding-top: 3px;
+}
+
+.info_area a {
+  color: #3ba3ff;
+}
+
+nav a:hover {
+  background: linear-gradient(0deg, rgba(95, 95, 95, 1) 0%, rgba(65, 65, 65, 1) 50%, rgba(75, 75, 75, 1) 51%, rgba(95, 95, 95, 1) 100%);
+}
+
+nav a:first-child {
+  line-height: 40px;
+  border-radius: 5px 0 0 5px;
+}
+
+.input-div {
+  width: auto;
+  display: flex;
+  align-items: center;
+  padding: 0 10px 0 10px;
+  background: linear-gradient(0deg, rgba(120, 120, 120, 1) 0%, rgba(95, 95, 95, 1) 50%, rgba(110, 110, 110, 1) 51%, rgba(120, 120, 120, 1) 100%);
+  border-radius: 0 5px 5px 0;
+}
+
+input {
+  outline: none;
+  border: none;
+  background: url("../assets/images/lupa.png") no-repeat 1%;
+  border-radius: 30px;
+  box-shadow: 0 1px #808080, 0 -1px #363636;
+  font-size: 14px;
+  color: white;
+  font-weight: bold;
+  width: 106px;
+  height: 19px;
+  padding-left: 20px;
+  transition: width 0.5s;
+}
+
+input:focus {
+  background: url("../assets/images/lupa_selected.png") no-repeat 1%, white;
+  width: 170px;
+}
+
+/* Text article */
+
+.text {
+  display: flex;
```

Comment:
CSS bad practices ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
 *  No repeating rule
    ```
      display: flex;
      align-items: center;
   ```